### ISSUE TITLE
Criar conjunto de dados com memória de cálculo agrupamentos DCMEFO

### ISSUE THREAD

**Issue by fjuniorr** [splor-mg/atividades#115]
Necessário para finalização de https://github.com/splor-mg/roperacional/issues/11. As definições em reunião com DCMEFO foram:

>    - Notificações para linhas não classificadas - ok
>    - Notificações para campo "outros" acima de 20% - ok. Conforme o operacional for exibindo, a planilha poderá ser aprimorada, detalhando agrupamentos que ficaram genéricos.
>    - Podemos utilizar o agrupamento antigo? Atualizar para nova planilha auxiliar definida pela DCMEFO.

**Comment by fjuniorr** [comment]
@hslinhares a gente vai implementar as notificações para fins de conhecimento mas para DCMEFO isso está finalizado, correto? Eles disseram que vão acompanhar linhas não classificadas e outros acima de 20% pelo painel do qlikview?

**Comment by hslinhares** [comment]
@fjuniorr  exato. Em conversa com o Pedro ele me disse que não precisam que o notifiquemos com esta informação e que farão a gestão da planilha por meio do painel qlikview. Mas ainda assim criamos um test apenas para notificação, mas vamos inserir numa função genérica que não para a execução, quando houver linhas não classificadas na execução.

https://github.com/splor-mg/aux-agrupamento-dcmefo/pull/3/files

**Comment by fjuniorr** [comment]
Vou considerar esse issue finalizado então e monitorar a questão da não paralisação da execução em https://github.com/splor-mg/cookiecutter-datapackage/issues/41 começando com esse nosso caso concreto.
